<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuizzGame</class>
 <widget class="QMainWindow" name="QuizzGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QuizzGame</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>311</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="welcome">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>100</y>
        <width>281</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_singIn">
         <property name="text">
          <string>Sing In</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_registerWellcome">
         <property name="text">
          <string>Register</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_wellcome">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>50</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Wellcome to Quizz Game!</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_status">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>220</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="login">
     <widget class="QGroupBox" name="SingIn">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>60</y>
        <width>248</width>
        <height>143</height>
       </rect>
      </property>
      <property name="title">
       <string>Sing In</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_username">
         <item>
          <widget class="QLabel" name="label_username_login">
           <property name="text">
            <string>Username</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_username_login</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_username_login"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_password">
         <item>
          <widget class="QLabel" name="label_password_login">
           <property name="text">
            <string>Password </string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_password_login</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_password_login"/>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_login">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="register_2">
     <widget class="QGroupBox" name="Register">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>60</y>
        <width>248</width>
        <height>143</height>
       </rect>
      </property>
      <property name="title">
       <string>Register</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_username_3">
         <item>
          <widget class="QLabel" name="label_username_register">
           <property name="text">
            <string>Username</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_username_login</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_username_register"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_password_3">
         <item>
          <widget class="QLabel" name="label_password_register">
           <property name="text">
            <string>Password </string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_password_login</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_password_register"/>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_register">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Register</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="game"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>597</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
